src/views/episodes-page/EpisodesPage.jsx

```diff
@@ -17,41 +17,49 @@ class EpisodesPage extends React.Component {
   }
 
   render() {
-    const { episodeTables, isRequesting } = this.props;
+    const { isRequesting } = this.props;
 
     return (
       <>
         <LoadingIndicator isActive={isRequesting} />
-        {episodeTables.map((model) => (
-          <div key={model.title}>
-            <Header as="h2">{model.title}</Header>
-            <Table>
-              <Table.Header>
-                <Table.Row>
-                  <Table.HeaderCell width={1}>Scene</Table.HeaderCell>
-                  <Table.HeaderCell>Episode</Table.HeaderCell>
-                  <Table.HeaderCell>Date</Table.HeaderCell>
-                  <Table.HeaderCell>Name</Table.HeaderCell>
-                </Table.Row>
-              </Table.Header>
-              <Table.Body>
-                {model.rows.map((row) => (
-                  <Table.Row key={row.episode}>
-                    <Table.Cell>
-                      <Image src={row.image} rounded={true} size="small" />
-                    </Table.Cell>
-                    <Table.Cell>{row.episode}</Table.Cell>
-                    <Table.Cell>{row.date}</Table.Cell>
-                    <Table.Cell>{row.name}</Table.Cell>
-                  </Table.Row>
-                ))}
-              </Table.Body>
-            </Table>
-          </div>
-        ))}
+        {this._renderTableGroup()}
       </>
     );
   }
+
+  _renderTableGroup() {
+    const { episodeTables } = this.props;
+
+    return episodeTables.map((model) => (
+      <div key={model.title}>
```

Comment:
what happens if there is another model with a same title?  these `key`s need to be unique

Reply:
In this case I know that titles will be unique because I create them in the selector.

Reply:
How would react and/or an implementor know that?  See also:

* https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md
* https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md (this is actually a counter-argument to my index suggestion)